src/features/query-bar/__tests__/FilterButton-test.js

```diff
@@ -184,29 +196,27 @@ describe('feature/query-bar/components/filter/FilterButton', () => {
     describe('createCondition()', () => {
         test('Should return a condition object if columns is non-empty', () => {
             const wrapper = getWrapper({ columns });
-            const conditionID = '11';
+            const conditionID = '12';
```

Comment:
This change is due to uniqueId generating a random number, in the case of this test, it is '12' and not '11' which I used before to pass in to ```createCondition```.

Reply:
We don't want tests to break randomly like this.

Alternatively, you can verify...
* That conditionId is non-empty
* columnId == 1
* Operator is EQUALS
* values == []